userguide/README.md

```diff
@@ -63,6 +63,7 @@ Using these toolkits, you can tackle a number of common scenarios:
 * [Sound classification](sound_classifier/README.md)
 * [Image similarity](image_similarity/README.md)
 * [Object detection](object_detection/README.md)
+* [One Shot object detection](one_shot_object_detection/README.md)
```

Comment:
I'm not sure the word "shot" should be capitalized here. The convention elsewhere in this list is to capitalize only the first letter of the first word.